src/Lib/Twig/TokenParser/Element.php

```diff
@@ -8,44 +7,42 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
 namespace WyriHaximus\TwigView\Lib\Twig\TokenParser;
 
+use Twig\Token;
+use Twig\TokenParser\IncludeTokenParser;
 use WyriHaximus\TwigView\Lib\Twig\Node\Element as ElementNode;
 
 /**
- * Class Element
+ * Class Element.
  * @package WyriHaximus\TwigView\Lib\Twig\TokenParser
  */
-class Element extends \Twig_TokenParser_Include
+class Element extends IncludeTokenParser
 {
-
     /**
      * Parse token.
      *
-     * @param \Twig_Token $token Token.
+     * @param \Twig\Token $token Token.
      *
-     * @return \Twig_NodeInterface|ElementNode
```

Comment:
`\Twig_NodeInterface` has been removed and should be replaced by `\Twig\Node\Node`.
However this method just returns `ElementNode`...